README

```diff
@@ -83,6 +100,17 @@ export USERLAND_ARCHIVES
 
         $ cd (component-dir) ; gmake publish
 
+    If you build several components, and some are required dependencies for the
```

Comment:
Sorry, haven't seen this. We removed references to ENCUMBERED as setting ENCUMBERED in ~/.profile breaks gmake components.mk in components dir.